keras_preprocessing/image/dataframe_iterator.py

```diff
@@ -51,7 +53,7 @@ class DataFrameIterator(BatchFromFilesMixin, Iterator):
             - `"sparse"`: 1D numpy array of integer labels,
             - `"input"`: images identical to input images (mainly used to
                 work with autoencoders),
-            - `"other"`: numpy array of `y_col` data,
```

Comment:
I would like to keep "other", or at least deprecate it.

Reply:
ah yeah, you're right. Let me think about it and propose if to deprecated or keep it, keeping in mind the future application for it. thanks!

Reply:
If 'other' is removed, is there any other way to do regression?

Reply:
I think we should keep it in some way, for regression as you mentioned for example. Let me think how exactly. The `other` word doesn't seem appropriate to me anymore, as it will be the use case for setting a single `y_col` column and returning that without any conversion. So it's very specific `other` I think refers to many types of output, perhaps `raw` instead of `other`? what do you think?

Reply:
done